libs/core/src/lib/static-utils/jwt-helper.ts

```diff
@@ -110,8 +114,7 @@ export class JwtHelper {
       return true
     } else {
       const date = this.getTokenExpirationDate(token)
-
-      if (date == null) {
+      if (date === null) {
```

Comment:
Does not the same anymore == null has true also for undefined values but === is strictly checking null